microcosm/opaque.py

```diff
@@ -63,7 +114,7 @@ def foo():
 
     def __init__(self, *args, **kwargs) -> None:
         self.service_name: Optional[str] = kwargs.pop("name", None)
-        self._store = ContextVar("store", default=dict(*args, **kwargs))
+        self._store = ContextVar("store", default=NormalizedDict(*args, **kwargs))
```

Comment:
@jamie-chang-globality please have a look whether it will play nicely with your initializer?

Reply:
I think this will be fine